ensure_no_std/src/main.rs

```diff
@@ -0,0 +1,21 @@
+#![no_std]
+#![no_main]
+
+use core::panic::PanicInfo;
+
+/// This function is called on panic.
+#[panic_handler]
+fn panic(_info: &PanicInfo) -> ! {
+    loop {}
+}
+
+#[no_mangle]
+pub extern "C" fn _start() -> ! {
+    loop {}
+}
+
+#[global_allocator]
+static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
+
+#[allow(unused_imports)]
+use stark_felt;
```

Comment:
use `starknet-types-core` and `starknet-types-rpc ` too.
Otherwise, they are not actually part of the binary and checked

Reply:
For now they're empty